source/partials/drupal-8-convert-to-composer.md

```diff
@@ -36,8 +36,7 @@ Set the Drupal core version, to ensure the site remains on Drupal 8 for now:
 
   ```bash{promptUser:user}
   composer require --no-update drupal/core-recommended:^8.9
-  composer require "composer/installers:^1.9"
-  composer update drupal/core* -W
+  composer require --dev drupal/core-dev:^8.9
```

Comment:
It would be more consistent to put this before the composer update and add `--no-update`, but not sure that's necessary. Does the `composer require` effectively update everything that was added `--no-dev` without updating other things that are not out of date?  Maybe not updating things that are not out of date isn't an issue in this context, since we're doing an initial composer conversion. So maybe this is okay. Have you considered / tested these issues?

Reply:
Yes. I ran both composer require with --no-update and the `composer update drupal/core* -W` and same result